realm/realm-library/src/main/java/io/realm/internal/async/QueryUpdateTask.java

```diff
@@ -100,6 +100,7 @@ public void run() {
             }
 
         } catch (Exception e) {
+            // TODO Should we just ignore BadVersionException?
```

Comment:
I think we should ignore it, any retry should be handled by the `HandlerController`